.deepsource.toml

```diff
@@ -0,0 +1,22 @@
+version = 1
```

Comment:
```suggestion
#
# Licensed to the Apache Software Foundation (ASF) under one or more
# contributor license agreements.  See the NOTICE file distributed with
# this work for additional information regarding copyright ownership.
# The ASF licenses this file to You under the Apache License, Version 2.0
# (the "License"); you may not use this file except in compliance with
# the License.  You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#

version = 1
```

Reply:
you should add a license info for the PR to pass.

but in general the decision to use deepsource.toml  library as a standard for code is a topic that needs to be considered  bc Superset already has a set of standards for python that are being validated at the pre-commit check

Reply:
@amitmiran137 DeepSource is a static analysis tool, which along with code style, also detects bug-risks, performance issues and security vulnerabilities.
It helps speeding up the code reviewing process.